packages/@cosmos-building-blocks/app-ec2/package.json

```diff
@@ -43,19 +43,19 @@
     "source-map-support": "^0.5.16"
   },
   "dependencies": {
```

Comment:
We need to fix this as peer, other wise it wont work with the rest of the lib, also version updating should happen across the libs not just for one lib.

Reply:
no actual peering yet - lets chat and update later